src/AxiomV2Addresses.sol

```diff
@@ -76,6 +95,25 @@ library AxiomV2Addresses {
         }
     }
 
+    /// @notice Returns the address of the AxiomV2QueryMock contract on the chain corresponding to `chainId`
+    /// @param targetChainId The chainId of the AxiomV2QueryMock contract
+    /// @param sourceChainId The chainId of the source chain
+    /// @return addr The address of the AxiomV2QueryMock contract
+    function axiomV2QueryMockCrosschainAddress(uint64 targetChainId, uint64 sourceChainId)
```

Comment:
separate from the other note above, but applicable to all of these crosschain address/deploy block functions: these functions will need to also take in an `isBlockhashOracle` and `bridgeId`, I believe

Reply:
Good catch